### PR TITLE
Add More Shortcuts

### PR DESCRIPTION
**Pull request by Daltz333**

### CODE REVIEW

**Review by ThadHouse — APPROVED** [review]

File: WPILibInstaller-Avalonia/ViewModels/InstallPageViewModel.cs [hunk 1]
```diff
@@ -652,8 +652,27 @@ private async Task RunShortcutCreator(CancellationToken token)
             shortcutData.IconLocation = Path.Join(frcHomePath, configurationProvider.UpgradeConfig.PathFolder, "wpilib-256.ico");
             shortcutData.IsAdmin = toInstallProvider.Model.InstallAsAdmin;
 
-            shortcutData.DesktopShortcuts.Add(new ShortcutInfo(Path.Join(frcHomePath, "vscode", "Code.exe"), $"FRC VS Code {frcYear}", $"FRC VS Code {frcYear}"));
-            shortcutData.StartMenuShortcuts.Add(new ShortcutInfo(Path.Join(frcHomePath, "vscode", "Code.exe"), $"FRC VS Code {frcYear}", $"FRC VS Code {frcYear}"));
+            // Add VS Code Shortcuts
+            shortcutData.DesktopShortcuts.Add(new ShortcutInfo(Path.Join(frcHomePath, "vscode", "Code.exe"), $"WPILib VS Code {frcYear}", $"FRC VS Code {frcYear}"));
```
Comment (sciencewhiz) [inline WPILibInstaller-Avalonia/ViewModels/InstallPageViewModel.cs:656]: Still have FRC?

**Review by sciencewhiz — CHANGES_REQUESTED** [review]
Start Menu Icons should be in the programs folder. image

I think it would also be more standard to install everything in to programs/WPILib 2021, rather then installing vscode at the top level and having a folder just for tools.

**Comment by sciencewhiz** [comment]
It would be nice if RobotBuilder used the RobotBuilder Icon, rather then the generic WPILib Icon. https://github.com/wpilibsuite/RobotBuilder/blob/master/src/main/resources/icons/robotbuilder.png

**Comment by Daltz333** [comment]
I can do the program thing. But the robotbuilder icon is not trivial for me as it's not downloaded in the installer.

**Comment by ThadHouse** [comment]
Make an issue for the icons and I can do it

**Comment by sciencewhiz** [comment]
It should go into Programs\2021 WPILib not 2021 Programs\2021 WPILib, and vscode needs to go there too.

image

**Review by sciencewhiz — APPROVED** [review]

**Pull request merged**